request/__init__.py

```diff
@@ -5,5 +5,6 @@
     'Kyle Fuller <kyle@fuller.li>',
     'Jannis Leidel (jezdez)',
     'krisje8 <krisje8@gmail.com>',
+    'Mariusz Felisiak <felisiak.mariusz@gmail.com>',
```

Comment:
Feel free to add yourself to the ` __copyright__` and LICENSE file too: https://github.com/django-request/django-request/blob/master/LICENSE#L1